docs/core/whats-new/whats-new-in-dotnet-core-2-0.md

```diff
@@ -0,0 +1,138 @@
+---
+title: What's new in .NET Core 2.0
+description: A description of the new feature found in .NET Core 2.0.
+keywords: .NET, .NET Core
+author: rpetrusha
+ms.author: ronpet
+ms.date: 08/07/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+---
+# What's new in .NET Core 2.0
+
+.NET Core 2.0 includes enhancements and new features in the following areas:
+
+- [Tooling](#tooling)
+- [Language support](#language-support)
+- [Platform improvements](#platform-improvements)
+- [API changes](#api-changes)
+- [Visual Studio integration](#visual-studio-integration)
+- [Documentation improvements](#documentation-improvements) 
+
+## Tooling
+
+**`dotnet restore` runs implicitly**
+
+In previous versions of the .NET Core, you had to run the [dotnet restore](../tools/dotnet-restore.md) command to download dependencies immediately after you created a new project with the [dotnet new](../tools/dotnet-new) command, as well as whenever you added a new dependency to your project. In .NET Core 2.0, `dotnet restore` runs implicitly when the `dotnet new` command executes, as well as if dependencies need to be updated when the `run`, `build`, and `publish` commands execute.
+
+## Language support
+
+In addition to supporting C# and F#, .NET Core 2.0 also supports Visual Basic.
+
+**Visual Basic**
+
+With version 2.0, .NET Core now supports Visual Basic 2017. For example, to create a simple Visual Basic "hello world" application, do the following from the command line:
+
+1. Open a console window, create a directory for your project and make it the current directory.
+
+1. Enter the command `dotnet new console -lang vb`.
+
+   This creates a project file with a `.csproj` file extension, along with a Visual Basic source code file named `program.vb`. This file contains the source code to write the string "Hello World!" to the console window.
+  
+1.  Enter the command `dotnet run`. The [.NET Core CLI tools](../tools/) automatically compile and execute the application, which displays the message "Hello World!" in the console window.
+
+Visual Studio Preview 15.3 also includes templates for Visual Basic that let you create .NET Core console apps, class libraries, unit test projects, and xUnit test projects.
```

Comment:
I'd check what's the right branding for VS here. I'd guess is Visual Studio 2017 Update 3. And CLI also includes those templates.